app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -76,26 +81,36 @@ class ExplorationActivityPresenter @Inject constructor(
     this.internalProfileId = internalProfileId
     this.topicId = topicId
     this.storyId = storyId
+    this.explorationId = explorationId
     this.context = context
     this.backflowScreen = backflowScreen
-
-    if (getExplorationFragment() == null) {
-      val explorationFragment = ExplorationFragment()
+    if (getExplorationManagerFragment() == null) {
+      val explorationManagerFragment = ExplorationManagerFragment()
       val args = Bundle()
       args.putInt(
         ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY,
         internalProfileId
       )
-      args.putString(ExplorationActivity.EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY, topicId)
-      args.putString(ExplorationActivity.EXPLORATION_ACTIVITY_STORY_ID_ARGUMENT_KEY, storyId)
-      args.putString(
-        ExplorationActivity.EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY,
-        explorationId
-      )
-      explorationFragment.arguments = args
+      explorationManagerFragment.arguments = args
+      activity.supportFragmentManager.beginTransaction().add(
+        R.id.exploration_fragment_placeholder,
+        explorationManagerFragment,
+        TAG_EXPLORATION_MANAGER_FRAGMENT
+      ).commitNow()
+    }
+  }
+
+  fun loadExplorationFragment(result: StoryTextSize) {
```

Comment:
Nit: s/``result``/``storyTextSize``/ for consistency. It's not necessarily a 'result' here.

Reply:
Done.